### ISSUE TITLE
 ONNX exported TRT FP16 engine accuracy issues when running Polygraphy on GPU RTX2060

### ISSUE THREAD

**Issue by SolDogLi** [NVIDIA/TensorRT#2980]
## Description
I am attempting to optimize model inference using TensorRT.
I use polygraphy run model.onnx --trt --fp16 --onnxrt --atol 0.001,and get the issue:
FAILED | Difference exceeds tolerance (rel=1e-03 abs=1e-03)

## Environment

**TensorRT Version**:8.2.3

**NVIDIA GPU**:RTX2060

**NVIDIA Driver Version**:470.182.03

**CUDA Version**:10.4

**CUDNN Version**:8.1.0

Operating System:

Python Version (if applicable):

Tensorflow Version (if applicable):

PyTorch Version (if applicable):

Baremetal or Container (if so, version):

## Relevant Files

**Model link**:https://drive.google.com/drive/folders/16WmfS8lMcNLuyWJ0KS1Gn1oLEtP5_kPC?usp=sharing

## Steps To Reproduce
1、get onnx model and input json
2、polygraphy run model.onnx --onnxrt --trt --atol 1e-3 --rtol 1e-3  --verbose --load-inputs custom_inputs.json --fp16
3、[I]             Error Metrics: input.15
[I]                 Minimum Required Tolerance: elemwise error | [abs=0.020722] OR [rel=inf] (requirements may be lower if both abs/rel tolerances are set)
[I]                 Absolute Difference | Stats: mean=0.00023997, std-dev=0.00035038, var=1.2276e-07, median=0.00014275, min=0 at (0, 0, 41, 278), max=0.020722 at (1, 47, 90, 164), avg-magnitude=0.00023997
[I]                     ---- Histogram ----
                        Bin Range          |  Num Elems | Visualization
                        (0      , 0.00207) |   13029599 | ########################################
                        (0.00207, 0.00414) |      68326 | 
                        (0.00414, 0.00622) |       7431 | 
                        (0.00622, 0.00829) |       1358 | 
                        (0.00829, 0.0104 ) |        340 | 
                        (0.0104 , 0.0124 ) |         99 | 
                        (0.0124 , 0.0145 ) |         21 | 
                        (0.0145 , 0.0166 ) |         15 | 
                        (0.0166 , 0.0186 ) |          6 | 
                        (0.0186 , 0.0207 ) |          5 | 
[I]                 Relative Difference | Stats: mean=inf, std-dev=nan, var=nan, median=0.0014144, min=0 at (0, 0, 41, 278), max=inf at (0, 0, 37, 90), avg-magnitude=inf
[V]                     Could not generate histogram. Note: Error was: autodetected range of [0.0, inf] is not finite
[I]                     
[E]             FAILED | Difference exceeds tolerance (rel=0.001, abs=0.001)
[E]         FAILED | Mismatched outputs: ['input.15']
![image](https://github.com/NVIDIA/TensorRT/assets/33997872/f8b44e6c-4a41-4829-81c8-0692f1f50930)

**Comment by zerollzeng** [comment]
For FP16 I think it's expected, what's diff on FP32?

**Comment by SolDogLi** [comment]
> For FP16 I think it's expected, what's diff on FP32?

The diff on FP32 is Absolute Difference max=3.3379e-06.

If this error(max=0.020722) is in line with expectations for FP16, I would like to know how much the maximum error is in line with expectations.
I would like to know is there any criteria for judging the accuracy of FP16?

**Comment by zerollzeng** [comment]
> I would like to know is there any criteria for judging the accuracy of FP16?

The golden rule is use MAP over a validation set.
